src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -0,0 +1,114 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+    @Override
+    public Driver create(Driver driver) {
+        String createDriver = "INSERT INTO drivers (name, license_number) "
+                + "values(?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement createStatement =
+                         connection.prepareStatement(createDriver,
+                                Statement.RETURN_GENERATED_KEYS)) {
+            createStatement.setString(1, driver.getName());
+            createStatement.setString(2, driver.getLicenseNumber());
+            createStatement.executeUpdate();
+            ResultSet generatedKeys = createStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                driver.setId(generatedKeys.getObject(1, Long.class));
+            }
+            return driver;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert driver to DB"
+                    + driver, e);
+        }
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {
+        String get = "SELECT * FROM drivers WHERE is_deleted = FALSE AND id = ?;";
```

Comment:
```suggestion
        String getDriverQuery = "SELECT * FROM drivers WHERE is_deleted = FALSE AND id = ?;";
```
please use proper naming everywhere